_inc/lib/class.media-extractor.php

```diff
@@ -248,6 +248,11 @@ public static function extract_from_content( $content, $what_to_extract = self::
 						continue;
 					}
 
+					// Reject invalid URLs.
```

Comment:
Should we do this one step earlier, before we check if `$url['scheme']` is set to `data`?

Reply:
We could combine them. If the scheme isn't set OR the scheme is data, continue. Additionally, if no host is set, continue. 40f400292